src/routes/+layout.svelte

```diff
@@ -46,10 +47,10 @@
 				</li>
 			{/each}
 		</ul>
-	</nav>
-</Drawer>
+	</nav></Drawer
```

Comment:
I think this is also a chacnge that I made that you reverted... why? I Like the </Drawer> on its own line ^^